index.html

```diff
@@ -1057,6 +1057,47 @@ <h2>The <dfn>InteractionOutput</dfn> interface</h2>
     <section><h3>The <dfn>check data schema</dfn> algorithm</h3>
       To run the <a>check data schema</a> steps on |payload| and |schema:object|,
       <ol>
+        <li>
+          If |schema| is `null` or `undefined`, return `undefined`.
+        </li>
+        <li>
+          <!--
+            TODO: Should we just continue with the rest of the algorithm here?
+          -->
+          If |schema|.|default| is not |undefined| and |payload| is undefined,
+          let |payload| be the value of |schema|.|default|.
+        </li>
+        <li>
+          Let |const| be |schema|.|const|.
+          If |const| is not |undefined|,
+          <ol>
+            <li>
+              <!-- TODO: Should we already return here? -->
+              If |const| equals |payload|, return |payload|.
+            </li>
+            <li>
+              If |const| does not equal |payload|, throw {{TypeError}} and stop.
+            </li>
```

Comment:
I think the first condition could also be removed, as this way the other checks within the algorithm will also be applied to the payload (which would also uncover schemes that are unsatisfiable).